OrbitQt/main.cpp

```diff
@@ -140,10 +123,8 @@ static outcome::result<void> RunUiInstance(
   }());
   const auto& [ports, capture_path] = result;
 
-  options.asio_server_address =
-      absl::StrFormat("127.0.0.1:%d", ports.asio_port);
-  options.grpc_server_address =
-      absl::StrFormat("127.0.0.1:%d", ports.grpc_port);
+  ApplicationOptions options{
+      .grpc_server_address = absl::StrFormat("127.0.0.1:%d", ports.grpc_port)};
```

Comment:
Unfortunately not yet supported on the VS toolset that we use on the CI.

Reply:
Ah right, we have other code that does this but it's Linux only.